weave-js/src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -1,9 +1,9 @@
 import * as RadixDropdownMenu from '@radix-ui/react-dropdown-menu';
-import {Icon} from '@wandb/weave/components/Icon';
 import classNames from 'classnames';
 import React from 'react';
 import {twMerge} from 'tailwind-merge';
 
+import {Icon} from '../Icon';
```

Comment:
sorry to throw in this random change - I tried to run `yarn build-storybook` and it just could not handle the `@wandb/weave/components/Icon` import path for some reason

> Error: Could not load @wandb/weave/src/components/Icon (imported by ../../services/weave-python/weave-public/weave-js/src/components/DropdownMenu/DropdownMenu.tsx): ENOENT: no such file or directory, open '@wandb/weave/src/components/Icon'

After updating it to a relative path like this, it worked again. I'm too lazy to look into why atm /shrug

Reply:
Yes, we've seen that before and we have just been dealing with it exactly as you did.